scraper.js

```diff
@@ -107,6 +109,19 @@ async function execute() {
             finalResultsArray = scrapedResultsArray;
         }
 
+        const mergedResultsWithCoords = await getAllCoordinates(
+            finalResultsArray,
+            cachedResults
+        );
+        const timestamp = s3.getTimestampForFile();
+
+        if (WRITE_TO_FAUNA) {
+            await dbUtils.writeScrapedDataBatch(
```

Comment:
So I appreciate wanting to do fewer calls but this comes at a disadvantage; CVS will have results that hang around for probably over a minute waiting for Walgreens to finish before anything is written to the DB. I really like the idea of each scraper writing its result to the DB as soon as it finishes. Are you seeing issues with that approach already? Is that why you opted to pivot to having one big call at the end?

Reply:
I didn't do enough research on this before deciding to batch the calls, I just assumed there would be a significant operations overhead. So I'm refactoring to have each scraper write as it's finished. I'll keep around the tests in case we do want to batch more extensively in the future.

Reply:
I think this logic will actually be quite useful when we add the logic to read the data from the tables, so I think worth keeping in the code.